### PR TITLE
Htv3hc/feat/missing variable env

### PR DESCRIPTION
**Pull request by huavanthong**
Hi Thomas,

Could you help me review this pull request for this issue https://github.com/test-fullautomation/RobotFramework_AIO/issues/127 ?

Thank you and best regards,
Thong Hua

### CODE REVIEW

File: config/build/dpkg_build/postinst.sh [hunk 1]
```diff
@@ -171,6 +171,27 @@ if [ -d "${DLTCONNECTOR_PATH}" ]; then
    echo "sudo dpkg -i ${DLTCONNECTOR_PATH}${DLTCONNECTOR_NAME}"
 fi
 
+#
+# Remind user to install OpenJDK for Java
+#
+###############################################################################
```
Comment (test-fullautomation) [inline config/build/dpkg_build/postinst.sh:177]: Hi Thong,
this is a usability issue:
Most of the users will not know genpackagedoc as this is only required if you contribute to our repositories and write documentation which is plantuml based.
Therefore: Please don't ask, just try to install JDK. If it fails even no error message shall be created, we just proceed installation.
Thank you,
 Thomas

File: config/build/dpkg_build/set_robotenv.sh [hunk 2]
```diff
@@ -8,4 +8,19 @@ export RobotToolsPath=/opt/rfwaio/tools
...
+if [ -z "$JAVA_HOME" ]; then
+    # Find the path of OpenJDK using the update-alternatives command
+    JDK_PATH=$(update-alternatives --query javac | grep 'Value: ' | grep -o '/.*/bin/java')
+    
+    if [ -n "$JDK_PATH" ]; then
+        # Export JAVA_HOME if OpenJDK path is found
+        export JAVA_HOME=$(dirname $(dirname $JDK_PATH))
+        echo "JAVA_HOME set to $JAVA_HOME"
+    else
+        echo "No OpenJDK installation found."
```
Comment (test-fullautomation) [inline config/build/dpkg_build/set_robotenv.sh]: Hi Thong,
Please remove this this message (No OpenJDK installation found). This is only confusing, but OpenJDK is for normal users not required.
JAVA_HOME set to ... is O.K.
Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Thong,
please read my findings.
Thank you,
 Thomas

**Comment by huavanthong** [comment]
Hi Thomas,

I updated it. If there are any issues, please let me know.

Thank you and best regards,
Thong Hua

**Review by test-fullautomation — APPROVED** [review]
Hi Thong,
very good!
Thank you,
 Thomas

**Pull request merged**